doc_source/cni-upgrades.md

```diff
@@ -63,4 +65,75 @@ Any changes you've made to the plugin's default settings on your cluster can be
 
       ```
       kubectl apply -f aws-k8s-cni.yaml
-      ```
\ No newline at end of file
+      ```
+**Custom upgrade scenarios when CNI custom networking in place**
+
+The upgrade might be an involved process when custom networking is in place. While there are efforts underway to make this process seamless, the best practice is to consider backup, analyze, and update resources to minimize undesirable application downtime\. It is important to understand the exact steps might vary across different versions\. Below steps work best when only an image change is required.
```

Comment:
By involved, do you mean especially complex or lengthy?

Regarding
> While there are efforts underway to make this process seamless, the best practice is to consider backup, analyze, and update resources to minimize undesirable application downtime\.

I'm a bit confused by this sentence. Should I do the three actions in order? or is it three different suggestions?

Suggested:
> The below steps work best when only an image change is required.